docker/development/docker-compose.yml

```diff
@@ -27,14 +27,35 @@ services:
             - books
         environment:
             DATABASE_URI: postgresql://books:secret@books-db:5432/books
+            AWS_BOOKS_BUCKET_NAME: books
+            AWS_AUTHORS_BUCKET_NAME: authors
+            AWS_ENDPOINT_URL: http://localstack:4566
```

Comment:
For the future can we add variables for this kind of data. I mean to add a variable for aws_endpoit_url  or db uri

Reply:
Variables? What do you mean. Could you explain more